change/@ni-nimble-components-258183d7-7c0f-4208-978c-61b6089e7796.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Add Angular support for the nimble-card component",
```

Since the only change you made to nimble-components is updating the component-status story, you can mark the change type here as `none`.

```suggestion
  "type": "none",
  "comment": "Update the Angular status of the nimble-card component",
```

---

Cool, thanks, I was wondering about this. I'll update this in both PRs.